code/ethan/django_lab04/students_app/urls.py

```diff
@@ -0,0 +1,12 @@
+from .import views
+from django.urls import path, include
+from rest_framework.routers import DefaultRouter
+router = DefaultRouter()
+router.register('students', views.StudentViewSet, basename='students')
+
+app_name = 'students_app'
+
+urlpatterns = [
+    path('/', views.index, name='index'),
```

Comment:
![image](https://user-images.githubusercontent.com/47562501/204911464-59cffad5-2969-408f-b04c-3d0fb9cb929c.png)


This `/` needs to be removed and a slash `/` added to the path in `students.urls.py`:
* `path('/', views.index, name='index')` should be `path('', views.index, name='index')`
* `path('api', include('students_app.urls'))` should be `path('api/', include('students_app.urls'))`

## Current config results in the following:
![image](https://user-images.githubusercontent.com/47562501/204911674-4c610ff6-fbe3-41f6-a305-c814008d1f2d.png)
![image](https://user-images.githubusercontent.com/47562501/204911732-1aa88327-29e3-4e3c-ae2a-c278b3f78771.png)